### ISSUE TITLE
[Feature Request] disable darkreader on battery

### ISSUE THREAD

**Issue by lollilol** [darkreader/darkreader#13229]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

Yes

### Feature Request Description

as the title says, it automatically disables dark reader when you're on battery on your laptop

### Screenshots

![image](https://github.com/user-attachments/assets/af8bf34c-131e-4109-89be-f263efefa644)

**Comment by Rudxain** [comment]
This makes sense for back-lit displays, but not for OLED-like technologies

**Comment by lollilol** [comment]
i was just thinking about the CPU time being used to make the website black. I feel like the "Dynamic" Mode uses a lot of CPU.

On an OLED Panel it would still take those CPU resources, so you might still wanna disable dark reader on battery even if you have an OLED Panel.

**Comment by Rudxain** [comment]
Good point about CPU! I was also concerned about it when using DR on my Android with AMOLED, so I was using Static-mode. Then I decided to "pay the price" of Dynamic, for convenience.

It would be interesting if someone managed to get energy-use stats from multiple devices, to see if the display-savings outweigh the processing-cost (on average)
